docs/documentation/user.md

```diff
@@ -1,36 +1,31 @@
-# User documentation
+# Documenting Zulip's features
```

Comment:
Do we want to rebrand this to "Writing help center articles" or something similar to avoid having extra terminology for the same thing?

Reply:
Sounds good to me.

Reply:
Yeah, that was my first idea, and we can go that route. I was trying to have this section's title be consistent with the other two documentation tutorials ("Documenting an integration" and "Documenting REST API endpoints"). Which all line up in the sidebar titles as you can see in the screenshot just above ...

Reply:
Here's a quick comparison of with "Writing help center articles" as the article title in the sidebar. I think it works, but perhaps we want another title for the sub-heading "Writing documentation"?
![Screenshot from 2022-01-06 15-59-21](https://user-images.githubusercontent.com/63245456/148402811-2eb577b6-1f45-44f5-82a1-9cb7305e6b3a.png)